fem/libceed/ceed.cpp

```diff
@@ -256,6 +267,28 @@ void CeedAssembleDiagonalPA(const CeedData *ceedDataPtr,
 #endif
 }
 
+void RemoveCeedBasisAndRestriction(const FiniteElementSpace *fes)
+{
+#ifdef MFEM_USE_CEED
+   for (auto entry : internal::ceed_basis_map)
+   {
+      if (std::get<0>(entry.first)==fes)
+      {
+         CeedBasisDestroy(&entry.second);
+         internal::ceed_basis_map.erase(entry.first);
+      }
+   }
+   for (auto entry : internal::ceed_restr_map)
+   {
+      if (std::get<0>(entry.first)==fes)
+      {
+         CeedElemRestrictionDestroy(&entry.second);
+         internal::ceed_restr_map.erase(entry.first);
+      }
+   }
```

Comment:
Does it make sense to use `auto &entry : ` here instead of `auto entry : `?

Reply:
I don't think it makes a big difference in this case, but it doesn't hurt.

Reply:
Yeah I think the copy should be very cheap anyway

Reply:
I wish we could use decomposition declaration:
```c++
for (auto &[key, value] : internal::ceed_basis_map)
```